doc/neps/nep-0028-deprecation_policy.rst

```diff
@@ -0,0 +1,184 @@
+     months prior to our planned release date or oldest that supports the
+     minimum Python version (whichever is higher)
+
+   The minimum supported version of ``CPython`` will be set to
+   ``python_requires`` in ``setup`` and all supported versions of
+   Python will be in the test matrix and have binary artifacts built
+   for releases.
+
+   We will bump the minimum Python and NumPy versions as we can on
+   every minor and major release, but never on a patch release.
+
+For other dependencies, adopt similar time windows 24 months shorter.
+
+
+Backward compatibility
+----------------------
+
+No issues other than the intentional dropping of old version of
+``CPython`` and dependencies.
+
+
+Alternatives
+------------
+
+Ad-Hoc
+~~~~~~
+
+Projects could on every release evaluate if they want to increase
+the minimum version of Python supported.  While this is a notionally
+simple policy, it makes it hard for downstream users to predict what
+the future minimum versions will be.  As there is no objective threshold
+to when the minimum version should be dropped, it is easy for these
+discussions to devolve into bike shedding and acrimony.
+
+
+All Python Software Foundation supported versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is a very clear and conservative approach.  However, it means that
+there is 4 year lag between when new language features come into the
+language and when the projects are able to use them.  Additionally,
+for projects that have a significant component of compiled extensions
+this requires building many binary artifacts for each release.
+
+For the case of NumPy, many projects carry workarounds to bugs that
+are fixed in subsequent versions of NumPy.  Being proactive about
+increasing the minimum version of NumPy will allow downstream
+packages to carry fewer version-specific patches.
+
+
+
+Default version on Linux distribution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The policy could be support the version of Python that ships by
+default in the latest Ubuntu LTS or CentOS/RHEL release.  However, we
+would still have to standardize across the community which
+distribution we are following.
+
+By following the versions supported by major Linux distributions, we
+are giving up technical control of our projects to external
+organizations that may have different motivations and concerns than we
+do.
+
+N minor versions of Python
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Given the current release cadence of the CPython, the proposed time
+(42 months) is roughly equivalent to "the last two" CPython minor
+versions.  However, if CPython changes their release cadence, any rule
+based on the number of minor releases will need to be changed.
+
+
+Time window on the X.Y.1 CPython release
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As the first bug fix release is typically a few months after the
+initial release, you can achieve the same name effect by making the
+window longer which is easier to explain.
```

Comment:
Is "same **name** effect" a typo? 
"X.Y.1 release" means "first bug fix release"?
"You can achieve the same name effect by making the window longer" - here, "window" refers to the window relative to the minor release (rather than "window" relative to the first bug fix release), and that it is easier to explain a window relative to a minor release than it is to explain a window relative to a first bug fix release?
Maybe the distinction is not so easy to explain! I'm afraid I had to read this several times before I understood.

Reply:
Tried to re-word this, hopefully it is better now?